raspiraw.c

```diff
@@ -560,45 +607,54 @@ static void buffers_to_rawcam(RASPIRAW_CALLBACK_T *dev)
 	while ((buffer = mmal_queue_get(dev->rawcam_pool->queue)) != NULL)
 	{
 		mmal_port_send_buffer(dev->rawcam_output, buffer);
-		//vcos_log_error("Buffer %p to rawcam\n", buffer);
+		// vcos_log_error("Buffer %p to rawcam\n", buffer);
 	}
 }
 
 static void callback(MMAL_PORT_T *port, MMAL_BUFFER_HEADER_T *buffer)
 {
-	static int count = 0;
-	RASPIRAW_CALLBACK_T *dev = (RASPIRAW_CALLBACK_T*)port->userdata;
-	RASPIRAW_PARAMS_T *cfg = (RASPIRAW_PARAMS_T *)dev->cfg;
+	static int count	 = 0;
+	RASPIRAW_CALLBACK_T *dev = (RASPIRAW_CALLBACK_T *)port->userdata;
+	RASPIRAW_PARAMS_T *cfg	 = (RASPIRAW_PARAMS_T *)dev->cfg;
 	MMAL_STATUS_T status;
 
-	//vcos_log_error("Buffer %p returned, filled %d, timestamp %llu, flags %04X", buffer, buffer->length, buffer->pts, buffer->flags);
+	// vcos_log_error("Buffer %p returned, filled %d, timestamp %llu, flags
+	// %04X", buffer, buffer->length, buffer->pts, buffer->flags);
 	if (cfg->capture)
 	{
 
-		if (!(buffer->flags&MMAL_BUFFER_HEADER_FLAG_CODECSIDEINFO) &&
-                    (((count++)%cfg->saverate)==0))
+		if (!(buffer->flags & MMAL_BUFFER_HEADER_FLAG_CODECSIDEINFO) &&
+		    (((count++) % cfg->saverate) == 0))
 		{
 			// Save every Nth frame
 			// SD card access is too slow to do much more.
 			FILE *file;
 			char *filename = NULL;
-			if (create_filenames(&filename, cfg->output, count) == MMAL_SUCCESS)
+			if (create_filenames(&filename, cfg->output, count) ==
+			    MMAL_SUCCESS)
```

Comment:
Unnecessary line break. Do we have a max line length param that can be tweaked upwards? Even the kernel coding standard now allows up to 120 chars.